packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/templates/standaloneSearchBox.html

```diff
@@ -24,6 +24,7 @@
       without-submit-button={withoutSubmitButton}
       number-of-suggestions={numberOfSuggestions}
       textarea={textarea}
+      keep-filters-on-search={keepFiltersOnSearch}
```

Comment:
this is also something to cover in the tests:
```
      const searchBox = element.shadowRoot.querySelector(selectors.searchBox);

      expect(searchBox).not.toBeNull();
      expect(searchBox.keepFiltersOnSearch).toBe(expectedValue);
```

Reply:
oh yeah good point!